s3-lambda-comprehend-cdk-python/README.md

```diff
@@ -0,0 +1,127 @@
+# S3 -> Lambda -> Comprehend
+This patterns shows CDK deployment on how to leverage Amazon S3, AWS Lambda, Amazon Comprehend and Amazon DynamoDB to perform Batch Sentiment Analysis in a serverless fashion.
+
+## Architecture
+![Diagram](src/architecture.jpg)
+
+### What resources will be created?
+This CDK code will create the following:
+   - 1 S3 bucket (to hold the text for which sentiment analysis has to be done)
+   - 1 Lambda (to invoke the Comprehend API)
+   - 1 DynamoDB (to store the result of Sentiment Analysis)
+   - 1 IAM role (for the Lambda to invoke Comprehend and DynamoDB)
+
+## Requirements
+
+### Development Environment
+**Cloud 9**
+
+This code for this pattern is executed in AWS Cloud9 environment. The EC2 instance used is m5.large (8 GiB RAM + 2 vCPU).
```

Comment:
Why do I need a Cloud9 instance for this? Is it not possible to run this locally as well?

Reply:
The idea is to use Cloud9 to perform the CDK deployment. To use locally, user may have to use AWS SAM CLI to locally test the AWS CDK applications. 
I've updated the messaging.

Reply:
Sorry, maybe there is some confusion on my side. Could you reiterate why a Cloud9 environment is necessary and a local executing of `cdk synth && cdk deploy` is not advised? I've managed to deploy it from my local machine just fine. And how does the AWS SAM CLI relate to this?

Reply:
Apologies, my intention was to keep everything within AWS ecosystem (which includes the CDK synth and deploy). There is nothing blocking users from using local environment. It's just that I used a Cloud9 environment as per the convenience.

Users can also use AWS SAM CLI to deploy CDK apps. 
https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-cdk-testing.html

Reply:
Ok, thanks for the clarifications. I think it would be best to add a line about the possibility to do it locally as well. The Cloud9 instance will incur cost (even if minimal) and users might forget to delete it afterwards. On that note, I don't see a callout to terminate the instance at the end in the README.

Reply:
Thanks a lot for the observation.
Yes, I've added a comment showcasing capability to execute CDK deployment from local environment and towards the end to remind users to terminate the Cloud9 EC2 instance.